apps/system/js/app_transition_controller.js

```diff
@@ -83,8 +83,8 @@
   AppTransitionController.prototype._waitingForLoad = false;
   AppTransitionController.prototype.openAnimation = 'enlarge';
   AppTransitionController.prototype.closeAnimation = 'reduce';
-  AppTransitionController.prototype.OPENING_TRANSITION_TIMEOUT = 200;
-  AppTransitionController.prototype.CLOSING_TRANSITION_TIMEOUT = 200;
+  AppTransitionController.prototype.OPENING_TRANSITION_TIMEOUT = 1000;
+  AppTransitionController.prototype.CLOSING_TRANSITION_TIMEOUT = 1000;
```

Comment:
we were constantly hitting the timeout which meant that `TaskManagerUtils#waitForAppToClose` wasn't working properly.